GenerateIssuerIconAssets.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+command -v ag >/dev/null 2>&1 || { echo >&2 "I require The Silver Searcher (ag) but it's not installed. Aborting."; exit 1; }
+command -v sips >/dev/null 2>&1 || { echo >&2 "I require sips but it's not installed. Aborting."; exit 1; }
+
+get_name() {
+  # RegEx Explanation:
+  #   Positive Lookbehind -- (?<=\.\/IssuerIcons\/)
+  #     Asserts that the selected portion must be preceeded by ./IssuerIcons/
+  #   Pattern Match -- .*
+  #     Matches all characters, excluding newlines, of any length
+  #   Positive Lookforward -- (?=\.png)
+  #     Asserts that the selected portion must be followed by .png
+  echo $1 | ag --only-matching '(?<=\.\/IssuerIcons\/).*(?=\.png)'
+}
+
+write_json() {
+  # JSON copied from Xcode output
+  echo -n "{
+  \"images\" : [
+    {
+      \"idiom\" : \"universal\",
+      \"filename\" : \"${1}.png\",
+      \"scale\" : \"1x\"
+    },
+    {
+      \"idiom\" : \"universal\",
+      \"filename\" : \"${1}@2x.png\",
+      \"scale\" : \"2x\"
+    },
+    {
+      \"idiom\" : \"universal\",
+      \"filename\" : \"${1}@3x.png\",
+      \"scale\" : \"3x\"
+    }
+  ],
+  \"info\" : {
+    \"version\" : 1,
+    \"author\" : \"xcode\"
+  }
+}" > "${2}"
+}
+
+for file in ./IssuerIcons/*.png; do
```

Comment:
Do these relative paths rely on the current working directory from which the script is run? Do you think it would be possible to make them relative to the location of the script instead? Maybe using [`BASH_SOURCE`](http://mywiki.wooledge.org/BashFAQ/028)?

Reply:
In its current form the relative paths rely on the current working directory. I've implemented the other suggested changes, I'll take a look into this tomorrow – have to figure out how to make it work with the RegEx.

Reply:
Thanks! If it's a hassle, I'm not sure it's worth it!

Reply:
Maybe adding `cd "$(dirname "$0")"` at the top of the script to ensure the working directory is always as expected is easier.